pkg/apis/core/v1/defaults.go

```diff
@@ -349,6 +349,22 @@ func SetDefaults_HTTPGetAction(obj *v1.HTTPGetAction) {
 		obj.Scheme = v1.URISchemeHTTP
 	}
 }
+
+// SetDefaults_Namespace adds a default label for all namespaces
+func SetDefaults_Namespace(obj *v1.Namespace) {
+	// TODO, remove this in 1.22
```

Comment:
clarify the feature gate check is what should be removed in 1.22, not the whole block